public/doc/openapi-3-v1.0.0.json

```diff
@@ -1290,6 +1446,91 @@
         }
       }
     },
+    "/order_items/{order_item_id}/progress_messages/{id}": {
```

Comment:
@mkanoor is the IPP still enforced that requested not to use doubly nested subcollections?

Reply:
@syncrou @eclarizio  I think so we should try to limit nested ids. If there are 10 progress messages for an order item does it make sense to selectively delete them, shouldn't it always be all messages for the order item get deleted and discarded with a single API call.

Reply:
👍